index-nyc-2019.html

```diff
@@ -0,0 +1,254 @@
+          </p>
+        </div>
+        <div class="col-lg-6 col-md-6 col-sm-6">
+          <p>
+            Our experiences have shown that the theoretical can be put into practice with great success, and that topics once considered “advanced” can be made understandable and spread widely. For example, type systems, often considered complex,
+            can simplify our lives by teaching compilers to do some of our reasoning for us.
+          </p>
+          <p>C◦mp◦se :: Conference has been running in New York since 2015.</p>
+          <p>C◦mp◦se :: Melbourne, in partnership with <a
+         href="http://yowconference.com.au/">YOW! Australia</a>, brings this experience to Melbourne.</p>
+        </div>
+      </div>
+    </div><!-- /row -->
+  </div><!-- /container -->
+</div><!-- /aboutwrap -->
+
+<div class="sep portfolio" data-stellar-background-ratio="0.5" style="background-position: 50% -91.5px;"></div>
+
+<div id="portfoliowrap">
+  <div class="container">
+    <div class="row">
+      <h1>PAST CONFERENCES</h1>
+    </div>
+    <div class="row mt">
+      <div class="col-lg-5 col-md-5 col-sm-5 col-lg-offset-3 col-md-offset-3 col-sm-offset-3 topic">
+        <p class="text">
+          <b>C◦mp◦se :: Conference</b> has run since 2015.<br/>
+          Talk and proceedings from past conferences are online:
+          <br />
+          <br class="hidden-sm" />
+
+        </p>
+      </div>
+      <div class="col-lg-2 col-md-2 col-sm-2">
+        <p>
+          <a href="/2015">2015 NYC</a>
+          <br />
+          <a href="/2016">2016 NYC</a>
+          <br />
+          <a href="/2016-melbourne">2016 Melbourne</a>
+          <br />
+          <a href="/2017">2017 NYC</a>
+          <br />
+          <a href="/2017-melbourne">2017 Melbourne</a>
+          <br />
+          <a href="/2018-melbourne">2018 Melbourne</a>
+        </p>
+      </div>
+    </div>
+  </div>
+
+</div>
+
+<div class="sep coding" data-stellar-background-ratio="0.5" style="background-position: 50% -104px;"></div>
+
+<hr style="border:1px dotted transparent; " />
+
+<section id="who" name="who"></section>
+
+<div id="committeewrap">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-8 col-lg-offset-2 col-md-8 col-md-offset-2 col-sm-8 col-sm-offset-2 centered">
+          <h2> C◦mp◦se Organizers</h2>
+      </div>
+      <div class="col-lg-2 col-md-2 col-sm-2 col-xs-3 col-lg-offset-2 col-md-offset-2 col-sm-offset-2 ">
+        <img src="http://photos4.meetupstatic.com/photos/event/6/c/c/c/global_177507852.jpeg" class="img-responsive" />
+      </div>
+      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-9">
+        <p class="history">
+          C◦mp◦se is organized by the
+          <a href="http://www.meetup.com/NY-Haskell/">New York Haskell Users Group</a> and:
+          <br/>
+          <a href="/2019/organizers/">Gershom Bazerman, Doug Beardsley, Kat Chuang, Ryan Trinkle, Ashish Agarwal, Chelsea Betancourt, and Anastasia Bez.</a> <br/>
```

Comment:
Hmmm, this list is different than the one in the overall site index.

Reply:
I merged and then pushed a fix.

Reply:
Thanks, I'm sorry I got it wrong. It was a bit heady at the end.